lib/nexmo_markdown_renderer/filters/code_filter.rb

```diff
@@ -1,56 +1,60 @@
-class CodeFilter < Banzai::Filter
-  def call(input)
-    input.gsub(/(?!.*snippet)```code(.+?)```/m) do |_s|
-      config = YAML.safe_load($1)
-
-      if config['config']
-        configs = YAML.load_file("#{Rails.root}/config/code_examples.yml")
-        config = config['config'].split('.').inject(configs) { |h, k| h[k] }
+module Nexmo
+  module Markdown
+    class CodeFilter < Banzai::Filter
+      def call(input)
+        input.gsub(/(?!.*snippet)```code(.+?)```/m) do |_s|
+          config = YAML.safe_load($1)
+    
+          if config['config']
+            configs = YAML.load_file("#{Rails.root}/config/code_examples.yml")
```

Comment:
I don't think this block is needed, `#{Rails.root}/config/code_examples.yml`  does not even exist in `nexmo-developer`

Reply:
👍